ResourceLocator/PeopleMap.html

```diff
@@ -0,0 +1,764 @@
+    <script src="lib/calendar/fullcalendar.min.js"></script>
+	<link rel="stylesheet" href="https://code.jquery.com/ui/1.11.3/themes/smoothness/jquery-ui.css"/>
+
+<!-- End Needed libs for Calendar/JQuery datepicker -->
+<!-- Needed libs for Google Maps -->
+	<script type="text/javascript"  src="https://maps.googleapis.com/maps/api/js?key=YOUROWNKEY"></script>
+	<script type="text/javascript" src="https://www.google.com/jsapi"></script>
+	<script src="lib/js/CoveoGoogleMap.js"></script> 
+	<script src="lib/js/CoveoMarkerclusterer.js"></script> 
+<!-- End Needed libs for Google Maps -->
+<!-- Connect to Search end point -->
+  <script type="text/javascript">
+    $(function () {
+     Coveo.SearchEndpoint.endpoints['default'] = new Coveo.SearchEndpoint({
+      restUri: 'https://platform.cloud.coveo.com/rest/search',
+      accessToken: 'YOURTOKEN',
+      queryStringArguments: {
+        organizationId: 'CLOUDORGID'
+      }
+    });
+      Coveo.$('#search').coveo('init');
+	
+    });
+  </script>
+</head>
+<body id="search" class="CoveoSearchInterface" data-enable-history="true" data-design="new">
+
+  <style>
+    .shadow {
+      -webkit-box-shadow: 2px 2px 3px 0px rgba(0,0,0,0.75);
+      -moz-box-shadow: 2px 2px 3px 0px rgba(0,0,0,0.75);
+      box-shadow: 2px 2px 3px 0px rgba(0,0,0,0.75);
+    }
+    .calAvail{
+      background-color:#42d692;
+    }
+    .calAvailHalf {
+      background-color:#ee8a36;
+    }
+    .calNotAvail {
+      background-color:#f71313;
+	  color: yellow;
+    }
+    .ProposedHalf {
+      background-color:#ee8a36;
+    }
+    .Holiday {
+      border-color: green }
+    .Training {
+      border-color: red}
+    .Selected {
+      border-top: 2px;
+      border-bottom:2px;
+      border-color:black}
+    .NotSelected {
+      zoom: 1;
+      filter: alpha(opacity=30);
+      opacity: 0.3;
+    }
+    .calReserveHalf {
+      background-color:#ee8a36;
+    }
+    .calReserved {
+      background-color:#f71313;
+    }
+    .Calendar h2 {
+      font-size:1.1em;
+    }
+    .fc-bg tr td{
+      height: 20px;
+    }
+    .fc-content-skeleton tr {
+      height: 2px;
+    }
+	.fc-basic-view tbody .fc-row {
+       min-height: 1em; 
+	}
+	.fc-basic-view td.fc-day-number, .fc-basic-view td.fc-week-number span {
+		padding-right: 6px;
+		padding-top: 4px;
+	}
+	.fc-other-month { background-color:white;color:white;}
+	.coveo-main-section {
+	  max-width:none !important;
+	}
+	.coveo-search-section {
+		margin-top: 15px !important;
+		margin-bottom: 1px !important;
+}
+ .coveo-facet-column {
+    width: 20% !important;
+ }
+	.ui-widget{font-family:Arial,Helvetica,sans-serif;font-size:0.8em}
+  </style>
+  <script>
+  //--------------------------------------------
+  // ResultsRelated
+  // To show results related to the current result
+  //--------------------------------------------
+  
+  var Coveo;
+    (function (Coveo) {
+      var ResultsRelated = (function (_super) {
```

Comment:
Was this compiled from TypeScript?

It would be very useful to have the original files.